src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1324,4 +1324,14 @@ private AdsDataProvider() {
         }
     };
   }
+
+  @DataProvider
```

Comment:
I'm using the same pages for few tests, so maybe it's good to use it in one place